help/user-guide/sops.md

```diff
@@ -2,6 +2,58 @@
 title: Standard Operating Procedures (SOPs)
 ---
 
-## Standard Operating Procedures (SOPs)
-[Assay specific]
-SOPs are standard operating procedures which describe the protocol required to reproduce the assay. They can be in any format (word files, e-lab notebooks, code, annotated spreadsheets etc). Relevant SOPs can be linked directly to the assay via the dropdown menu.
\ No newline at end of file
+## What are SOPs?
+
+Standard Operating Procedures (SOPs), also known as Protocols, describe the steps required to reproduce an assay or part of an experimental workflow.
+
+SOPs can be uploaded in various formats (Word documents, PDFs, e-lab notebooks, code scripts, annotated spreadsheets, etc.) and stored as assets in SEEK. These assets can be linked to Assays and, in ISA-JSON compliant setups, also to Study Samples and Assay Samples to document the procedures applied.
+
+SEEK can be used in two modes:
+- As a general-purpose FAIR data management platform
+- As an ISA-JSON compliant system
+
+While SOP creation and management are largely the same in both modes, ISA-JSON compliance introduces one key requirement: the `SOP_type` field must be filled in to ensure valid exports.
+
+## SOPs in SEEK
+
+Regardless of the mode:
+
+- SOPs are treated as assets, similar to Data Files or Models.
+- They can be created, uploaded, versioned, and linked to Assays to document experimental methods.
+
+In ISA-JSON compliant environments, SOPs play a formal role in the experimental lineage:
```

Comment:
Not here in general SOP page; better include it in ISA-JSON specific pages if needed